scala-package/core/src/main/scala/org/apache/mxnet/Symbol.scala

```diff
@@ -955,9 +955,10 @@ object Symbol extends SymbolBase {
    * @return Symbol The created Symbol.
    */
   def arange(start: Float, stop: Option[Float] = None, step: Float = 1.0f,
-    repeat: Int = 1, name: String = null, dType: DType = Base.MX_REAL_TYPE): Symbol = {
+    repeat: Int = 1, inferRange: Boolean = false, name: String = null,
+    dType: DType = Base.MX_REAL_TYPE): Symbol = {
     val params = Map("start" -> start, "step" -> step,
-      "repeat" -> repeat, "dtype" -> dType.toString())
+      "repeat" -> repeat, "infer_range" -> inferRange, "dtype" -> dType.toString())
```

Comment:
@lanking520

Reply:
@taliesinb Same applies to what I have mentioned above